### DISCUSSION TITLE
Sugestões de Design Instrucional (Tour guiado)

### DISCUSSION THREAD

**Discussion by casssiahaluu** [frontendbr/forum#184] [Perguntas e Respostas]
Boa madrugada,

Estou desenvolvendo um sistema em Django, e gostaria de adicionar à ele aqueles tutoriais que destacam o botão ou o lugar, enquanto todo o resto fica desativo, induzindo o usuário a interagir com o sistema. Alguém tem alguma dica de como posso fazer algo do tipo? Obrigada desde já :+1:

**Comment by akfzambrana** [comment]
Bom dia, 

nós utilizamos o [IntroJS](http://introjs.com/) para fazer um tour pelas funcionalidades mais importantes dos portais que desenvolvemos, veja se serve para você :smile:

**Comment by casssiahaluu** [comment]
Nossa! Perfeito! Exatamente isso que precisava! Muito obrigada :) :+1:

**Comment by kandebonfim** [comment]
Legal. 👍  Só pra indexar aqui e ficar mais fácil de achar: Esse tipo de feature é comumente chamada de Onboarding ou Step-by-step demo / Guide

**Comment by casssiahaluu** [comment]
Muito obrigada pela dica. Estava com dificuldades em pesquisar por não saber o nome correto.

**Comment by kandebonfim** [comment]
@HaluDeCassia É tenso mesmo. kkkk
